src/node.h

```diff
@@ -209,8 +210,26 @@ NODE_EXTERN void Init(int* argc,
 class IsolateData;
 class Environment;
 
-NODE_EXTERN IsolateData* CreateIsolateData(v8::Isolate* isolate,
-                                           struct uv_loop_s* loop);
+class WorkerSupportingPlatform : public v8::Platform {
+ public:
+  virtual ~WorkerSupportingPlatform() { }
+  virtual void DrainBackgroundTasks(v8::Isolate* isolate) = 0;
+
+  // These will be called by the `IsolateData` creation/destruction functions.
+  virtual void RegisterIsolate(IsolateData* isolate_data,
+                               struct uv_loop_s* loop) = 0;
+  virtual void UnregisterIsolate(IsolateData* isolate_data) = 0;
+};
+
+// If `platform` is passed, it will be used to register new worker instances.
+// It can be `nullptr`, in which case creating new workers will not work.
```

Comment:
The word "creating new workers" is vague to me. Creating new workers in the isolate for which isolate data are created? or creating new workers whose isolate is the isolate for which isolate data are created?